default/AI/AIDependencies.py

```diff
@@ -134,4 +134,6 @@
     "FU_ZERO_FUEL": {FUEL_PER_TURN: FULL_FUEL},
     "SP_DISTORTION_MODULATOR": {},  # not supported yet
     "SH_ROBOTIC_INTERFACE_SHIELDS": {},  # not supported yet
+    PART_KRILL_SPAWNER: {STEALTH_MODIFIER: 60}
```

Comment:
Krill Spawner stealth was initially set to 60, but after some feedback from Mat I dropped it to 40.  Also, it looks to me like the Krill stealth is treated like it would stack with a regular stealth part, which it does not.